lib/Raven/Client.php

```diff
@@ -462,76 +412,51 @@ public function capture(array $payload)
         }
 
         $event = $this->callMiddlewareStack($event, static::isHttpRequest() ? ServerRequestFactory::fromGlobals() : null, isset($payload['exception']) ? $payload['exception'] : null, $payload);
+        $event = $this->sanitize($event);
 
-        $payload = $event->toArray();
-
-        $this->sanitize($payload);
-
-        if (!$this->storeErrorsForBulkSend) {
-            $this->send($payload);
-        } else {
-            $this->pendingEvents[] = $payload;
-        }
+        $this->send($event);
 
         $this->lastEvent = $event;
 
-        return $payload['event_id'];
+        return str_replace('-', '', $event->getId()->toString());
```

Comment:
Why we omit the dashes?

Reply:
Because as per [SDK specs](https://docs.sentry.io/clientdev/attributes/) the ID of an event is an UUID without dashes. Unfortunately we must do this string-replace everywhere we use the Ramsey UUID object

>Hexadecimal string representing a uuid4 value. The length is exactly 32 characters (no dashes!)

Reply:
Maybe then it's better if we do that at the start, wrapping the Ramsey lib call?

Reply:
I was experimenting with a custom codec, which should be the correct way to format an UUID using the Ramsey lib, but this required a bunch of other things like the ability to change the factory and the changes were too much and out of scope for this PR. I would like to address this problem better in the future